ComputeDemo/ComputeAlphaFinePointsChecks.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * Copyright 2020 Google LLC
```

Comment:
Rumour has it that 2020 is (thankfully) over now.

Reply:
👍